macosx/Controller.mm

```diff
@@ -2707,6 +2707,12 @@ - (void)applyFilter
     }
     BOOL const filterTracker = searchStrings && [[fDefaults stringForKey:@"FilterSearchType"] isEqualToString:FILTER_TYPE_TRACKER];
 
+    std::atomic<int32_t> active{0}, downloading{0}, seeding{0}, paused{0};
```

Comment:
FYI, `__block` somehow is not working here.
```
error: call to implicitly-deleted copy constructor of 'std::atomic<int32_t>' (aka 'atomic<int>')
     __block std::atomic<int32_t> active{0};
```
I think it's because of underlying implementation of Obj-C Blocks. [Clang docs](https://clang.llvm.org/docs/Block-ABI-Apple.html)